CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-Thank you for contributing to the docs! We have a few guidelines which will allow your PR to pass our CI checks and successfully merge.
+Thank you for contributing to the docs! We have a few guidelines which will allow your patch: pull request (PR) to pass our CI checks and successfully merge.
```

Comment:
```suggestion
Thank you for contributing to the docs! We have a few guidelines which will allow your pull request (PR) to pass our CI checks and successfully merge.
```

Reply:
Here's the specific change I am suggesting. Hope this clears it out what need to be changed in your PR